docs/_articles/en/einstein/einstein-inline.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Inline Automatic Completions
+lang: en
+---
+
+## Overview
+
+Use AI-based autocomplete to accept suggestions for repetitive, boilerplate code right in your editor.
```

Comment:
The inline completions feature doesn't discriminate as to which code it tries to auto-complete.  :) - Could we replace "repetitive, boilerplate" here?  

maybe something like
"Use AI-based autocomplete to accept suggestions for code _as you write it_ right inside your editor."